frontend/package.json

```diff
@@ -27,6 +27,7 @@
     "axios": "^0.27.2",
     "graphql": "^16.5.0",
     "i18next": "^21.8.10",
+    "lodash": "^4.17.21",
```

Comment:
As much as I love lodash (and I do), there are no usages at the moment, so let's remove this

Reply:
done